src/app/page.tsx

```diff
@@ -1,13 +1,15 @@
-import Image from 'next/image'
-import styles from './page.module.css'
+import Image from "next/image";
+import styles from "./page.module.css";
```

Comment:
we need to set up the linter/formatter for keeping the styles consistent between us